src/pages/login/login.ts

```diff
@@ -31,12 +32,17 @@ export class LoginPage {
     };
 
     this.isLoading = false;
+    this.isChecked = true;
```

Comment:
please make the var name more understandable .. `isChecked` is very vague ... What is checked ? In future we may have even a remember me checkbox ...